### PR TITLE
Removing border from facet section, changing spacing of heading and button

### PR DESCRIPTION
**Pull request by hudajkhan**
Addresses #1277 

To remove the border around the facet section for the mobile view, we override the border value for .facets-toggleable-md.  To vertically align the heading and the button, we remove some padding and margin values that were pushing the text further below.

### CODE REVIEW

File: app/assets/stylesheets/earthworks.css [hunk 1]
```diff
@@ -285,3 +285,14 @@ label.toggle-bookmark {
   --bl-facet-active-item-color: rgb(var(--stanford-palo-alto-rgb));
   border-color: rgb(var(--stanford-palo-alto-rgb));
 }
+
+/* Override mobile view for facets section in Blacklight */
+@media (max-width: 991.98px) {
+  .facets-toggleable-md {
+    border: none;
+    padding-top: 0px;
```
Comment (dnoneill) [inline app/assets/stylesheets/earthworks.css:293]: I would change this to 
.facets {
  --bl-facets-smallish-border: none;
  --bl-facets-smallish-padding: 0;
}

And I think you can get rid of the @media

Comment (hudajkhan) [reply app/assets/stylesheets/earthworks.css]: I changed the above, but couldn't figure out how to get facets-heading to change in a similar fashion (I tried --bl-facets-smallish-margin-bottom but the .facets-heading margin-bottom value of 0.5 rem kept overriding that)

File: app/assets/stylesheets/earthworks.css [hunk 2]
```diff
@@ -317,3 +317,29 @@ label.toggle-bookmark {
...
+    margin-bottom: 0;
+  }
+}
+
+/* Hide/show button on mobile */
+.facets-header .navbar-toggler {
+  --bl-facets-smallish-border: none;
+  --bs-navbar-toggler-border-color: var(--stanford-digital-blue);
+  --bs-navbar-color: var(--stanford-digital-blue);
+  color: var(--stanford-digital-blue);
```
Comment (jcoyne) [inline app/assets/stylesheets/earthworks.css:339]: is this `color` necessary? This rule should apply:
```
.navbar-toggler {
...
	color: var(--bs-navbar-color);
}
```

File: app/assets/stylesheets/earthworks.css [hunk 3]
```diff
@@ -317,3 +317,29 @@ label.toggle-bookmark {
...
+}
+
+.facets-header .navbar-toggler:hover {
+  color: var(--stanford-digital-blue-dark);
```
Comment (jcoyne) [inline app/assets/stylesheets/earthworks.css:343]: We should make this settable in blacklight: https://github.com/projectblacklight/blacklight/blob/69373f202753f04ec6aca179fdf8fa01248a9edf/app/assets/stylesheets/blacklight/_facets.scss#L22-L25

File: app/assets/stylesheets/earthworks.css (hunk 2 above)
Comment (hudajkhan) [reply app/assets/stylesheets/earthworks.css:339]: For whatever reason, it doesn't work with just the "--bs-navbar-color" definition.The "color" property is set explicitly elsewhere, so --bs-navbar-color isn't the only thing determining the color.

File: app/assets/stylesheets/earthworks.css (hunk 3 above)
Comment (hudajkhan) [reply app/assets/stylesheets/earthworks.css:343]: Do you mean, for example, make a variable specifically for the hover color in Blacklight, so we can just override that  variable value?

Comment (jcoyne) [reply app/assets/stylesheets/earthworks.css:343]: Exactly that.

**Comment by hudajkhan** [comment]
To make the rebase to main easier, and to get this moving along, I am going to close this PR and open up two new ones: 
one for the outline around the facet section and vertical alignment and one for changing the button to a primary outline button.

**Pull request closed without merging**